### ISSUE TITLE
Will there be an NPM package?

### ISSUE THREAD

**Issue by mcintyre94** [slipHQ/run-wasm#14]
More of a future looking question than an issue really, but I'm just wondering if the plan is to eventually have an NPM package for run-wasm? It seems like currently it'll be quite tricky to integrate this into an existing repo and keep up to date with improvements here. Would the eventual plan be to just be able to `npm install run-wasm` or is there something blocking that?

**Comment by kennethcassel** [comment]
Yeah we intend do have an npm package! Haven't gotten around to setting it up yet though. Need to look into it :) 

I'll keep this issue open until we have one
